lib/ClangImporter/ImportDecl.cpp

```diff
@@ -9092,7 +9093,51 @@ class SwiftifyInfoPrinter {
     out << "]";
   }
 
+  void printAvailability() {
+    printSeparator();
+    out << "availability: [";
+    if (!SwiftContext.LangOpts.Target.isOSDarwin() || SwiftContext.LangOpts.Target.isDriverKit()) {
```

Comment:
@DougGregor Does this make sense? Is `@available` a thing for driverKit or non-Darwin OSs?